### ISSUE TITLE
[Question] if we use onRedirectResult, how can we retreive of the redirect resulted in an error instead of results

### ISSUE THREAD

**Issue by maysam** [prescottprue/react-redux-firebase#646]
**Do you want to request a *feature* or report a *bug*?**  

(If this is a *usage question*, please **do not post it here**—post it on [gitter](https://gitter.im/redux-firebase/Lobby). If this is not a “feature” or a “bug”, or the phrase “How do I...?” applies, then it's probably a usage question.)

**What is the current behavior?**

**If the current behavior is a bug, please provide the steps to reproduce and if possible a minimal demo of the problem via [codesandbox](https://codesandbox.io/) or similar.**

**What is the expected behavior?**

**Which versions of dependencies, and which browser and OS are affected by this issue? Did this work in previous versions or setups?**

**Comment by prescottprue** [comment]
What method are you calling that causes the redirect result? For example, if you are calling `login`, then the error should be passed back to the `catch` of that promise.

**Comment by maysam** [comment]
@prescottprue this is true with popup calls, but when we come back from a redirect, the promise is not there no more

**Comment by prescottprue** [comment]
@maysam [The error is dispatched to state by default](https://github.com/prescottprue/react-redux-firebase/blob/master/src/actions/auth.js#L423-L430), so you can connect to that piece of state and display a message or redirect if that error exists. Alternatively, you should be able to set `getRedirectResult` directly on your Firebase instance - [that is is what is being done internally](https://github.com/prescottprue/react-redux-firebase/blob/master/src/actions/auth.js#L423-L430), but doing it yourself outside of react-redux-firebase should also yield the success/error promise.

**Comment by maysam** [comment]
@prescottprue I know about this internal version, but when we have onRedirectResult in the config, it's good to also be able to handle the error in that place too

**Comment by prescottprue** [comment]
@maysam I'll change this to a ticket to track that as a feature
